README.md

```diff
@@ -1,29 +1,55 @@
 # GOV.UK CDN config
 
-Configuration for GOV.UK's content delivery network (CDN).
+Configuration for GOV.UK's content delivery network (CDN). You can read more about [how GOV.UK CDN works](https://docs.publishing.service.gov.uk/manual/cdn.html) in the developer docs.
 
-Configures the [Fastly CDN](https://fastly.com) from version-controllable VCL and
-YAML files, using Fastly's [API](https://docs.fastly.com/api/).
+⚠️ This repo has some [documented tech debt](https://trello.com/c/y6MIgxjp). It doesn't conform to GOV.UK standards, and lacks sufficient testing. Approach code changes with caution.
```

Comment:
Is this a bit alarmist? There's a fine line between caution vs. putting people off wanting to do something...

Reply:
👍 that's a great point, I've rewritten it to be explicit about what I mean by "approach code changes with caution" (which is manual testing).